src/domains/charts/providers/menuGroup/menuGroup.js

```diff
@@ -0,0 +1,90 @@
+import React from "react"
+import styled, { css } from "styled-components"
+import { Text, Flex, getColor } from "@netdata/netdata-ui"
+import { MenuItemContainer } from "domains/charts/providers/subMenu"
+import { MenuGroupConsumer } from "./context"
+
+const MenuGroupLabel = styled(Text).attrs({ color: "border" })`
+  font-weight: "500";
```

Comment:
Why don't we use the prop strong in here?

Reply:
I copied the styling of the sidebar as it is, it has font-weight 500 for this list item...